src/test/java/gov/healthit/chpl/aqa/pageObjects/InactiveCertificatesPage.java

```diff
@@ -0,0 +1,18 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class InactiveCertificatesPage definition.
+ */
+public final class InactiveCertificatesPage extends BasePage {
+    private InactiveCertificatesPage() {}
+
+    public static WebElement inactiveCertificatesPageTitle(final WebDriver driver) {
+        return driver.findElement(By.xpath("/html/body/ai-compare-widget/div[2]/ui-view/div/div[1]/div/h1/ng-transclude/ai-title"));
```

Comment:
Why does this locator use the `ai-title` at the end, but the SedInfoFor2015Page.java doesn't?

Also, all of the collections pages have the same structure; could they be combined into a single "CollectionsPage.java" class?

Even the steps are almost the same; is there any way to reduce that duplication? Maybe with just a CollectionsBaseSteps.java that extends from Base.java, and has that step for "navigate to a collections page", and then have an InactiveCertificatesSteps that extends from the Collections steps class if there are specific steps needed for those activities